config/locales/en.yml

```diff
@@ -115,6 +115,10 @@ en:
             relationship:
               blank: Tell us about the referee’s relationship to you and how long they’ve known you
               too_many_words: Relationship to referee must be %{count} words or fewer
+        application_choice:
+          attributes:
+            status:
+              invalid_transition: The application is not ready for that action
```

Comment:
I think this is fine. We should include the flowchart (or the relevant bit) in the docs, maybe?

Reply:
I'll add an item to the docs checklist for adding the flowchart and look at that in the follow up PR